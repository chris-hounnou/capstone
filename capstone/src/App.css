.App
{
  display: flex;
  align-items: center;
  flex-direction: column;
  justify-content: space-between;
  width: 100vw;
  min-height: 100vh;
  background-color: rgb(80, 164, 164);
}
