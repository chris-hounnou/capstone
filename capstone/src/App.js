import React from 'react';
import { BrowserRouter, Routes} from 'react-router-dom';
import styles from "./App.module.css"
import NavigationBar from './components/NavigationBar';
import AppRoute from './routes/AppRoute';
import backgroundImage from './services/backgroundImage.jpg';

function App() {
  const backgroundStyle = {
    backgroundImage: `url(${backgroundImage})`
  };

  return (
<BrowserRouter>

    <div className={styles["app"]} style={backgroundStyle}>
          <header className={styles["app__header"]}>
            <NavigationBar/>
          </header>

          <main className={styles["app__main"]}>
            <AppRoute/>
          </main>

          <footer className={styles["app__footer"]}>
            
          </footer>
        </div>
        
        </BrowserRouter>
      
    );
}

export default App;
