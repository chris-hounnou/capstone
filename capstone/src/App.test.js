import { render, screen } from '@testing-library/react';
import App from './App';

test('it renders without crashing',() => {
render(<App/>)
})

test("matches snapshot", function() {
  const {asFragment} = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});