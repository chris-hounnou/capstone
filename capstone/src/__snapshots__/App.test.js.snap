// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="app"
    style="background-image: url(backgroundImage.jpg);"
  >
    <header
      class="app__header"
    >
      <nav
        class="navigationBar"
      >
        <a
          activeclassname="active"
          class="navLink"
          href="/"
        >
          Home
        </a>
        <a
          activeclassname="active"
          class="navLink"
          href="/Characters"
        >
          Characters
        </a>
        <a
          activeclassname="active"
          class="navLink"
          href="/Houses"
        >
          Houses
        </a>
      </nav>
    </header>
    <main
      class="app__main"
    >
      <div
        class="Home"
      >
        <h1
          class="title"
        >
          Welcome To The Land of Ice and Fire!
        </h1>
        <p>
          This is the Home page to the Game of Thrones App! navigate to other characters and lands in the features above.
        </p>
      </div>
    </main>
    <footer
      class="app__footer"
    />
  </div>
</DocumentFragment>
`;
